tests/integration_tests/performance/test_cpu_template_benchmark.py

```diff
@@ -24,16 +22,16 @@
 
 BASELINES = {
     "Intel": {
-        "deserialize": {"target": 0.025, "delta": 0.02},  # milliseconds
-        "serialize": {"target": 0.015, "delta": 0.02},  # milliseconds
+        "deserialize": {"max_target": 0.05},  # milliseconds
```

Comment:
Any reason this is different from main where it is `0.02`?

Reply:
The main branch has other changes which have affected the time for JSON operations. Faster/slower or just higher variance, I am not sure, but I have increased the max target to be safe. A target of 50 microseconds is still fast enough to not be a problem.